src/DF Envimet Simple Force by EPW.py

```diff
@@ -0,0 +1,124 @@
+# DF Envimet Simple Force by EPW
+#
+# Ladybug: A Plugin for Environmental Analysis (GPL) started by Mostapha Sadeghipour Roudsari
+# 
+# This file is part of Ladybug.
+# 
+# Copyright (c) 2013-2018, Antonello Di Nunzio <antonellodinunzio@gmail.com> 
+# Ladybug is free software; you can redistribute it and/or modify 
+# it under the terms of the GNU General Public License as published 
+# by the Free Software Foundation; either version 3 of the License, 
+# or (at your option) any later version. 
+# 
+# Ladybug is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of 
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the 
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with Ladybug; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+
+"""
+This component let you force climate condition of the simulation. Data of this component comes from EPW file.
+Basically, it extract 24 values of temperature and relative humidity by using DOY.
+-
+It is a good element to find a relation between calculation with EnergyPlus and calculation with ENVI_MET.
+-
+Connect "simpleForce" to ...
+-
+Provided by Dragonfly 0.0.03
+    
+    Args:
+        _epwFile:  An .epw file path on your system as a string.
+        _DOY: The day of the year. Connect the output from "Ladybug_DOY_HOY".
+        _HOY: The hour of the year. Connect the output from "Ladybug_DOY_HOY".
+    Returns:
+        readMe!: ...
+        dryBulbTemperature: The dry bulb temperature values [K] from selected day.
+        relativeHumidity: The relative humidity values [%] from selected day.
+        startDate: Start date for simulation settings.
+        startTime: Start time for simulation settings.
+"""
+
+ghenv.Component.Name = "DF Envimet Simple Force by EPW"
+ghenv.Component.NickName = 'DFenvimetSimpleForceByEPW'
+ghenv.Component.Message = 'VER 0.0.03\nJUL_08_2018'
+ghenv.Component.IconDisplayMode = ghenv.Component.IconDisplayMode.application
+ghenv.Component.Category = "Dragonfly"
+ghenv.Component.SubCategory = "6 | Utility Envimet"
```

Comment:
How many Envi-Met tabs are there?  I was thinking that the Envi-met capabilities would be like the UWG capabilities in which case there is just one tab for all of Envi-met.  I worry that people might get confused about which components are for which workflows otherwise.